src/ansible/Makefile

```diff
@@ -0,0 +1,23 @@
+all:
```

Comment:
Don't merge. Still some open issues:
1. Use ubuntu user for everything instead of creating cadizm?
2. Nginx running, but requests timing out (instance not powerful enough?)